typings/main/ambient/node/node.d.ts

```diff
@@ -0,0 +1,2196 @@
+// Compiled using typings@0.6.8
```

Comment:
Do you intentionally have 2 node.d.ts files?  (one under typings/browser and one under typings/main)